loader_hub/memos/base.py

```diff
@@ -31,21 +32,16 @@ def load_data(self, params: Optional[Any] = 101) -> List[Document]:
         import requests
         
         documents = []
-
         realUrl = self._memoUrl
+        
         if ~params:
             params = {}
-            realUrl += "/all"
+            realUrl = urljoin(self._memoUrl, "/all")
 
         req = requests.get(realUrl, params)
-        
-        if ~req.ok:
```

Comment:
Err I meant if request is not ok, `raise ValueError("Your Memo URL is not valid")` or something like that

Reply:
Ok